packages/react/tsup.config.ts

```diff
@@ -0,0 +1,62 @@
+import { default as fs } from 'fs-extra'
+import { defineConfig } from 'tsup'
+
+import { dependencies, peerDependencies } from './package.json'
+import path from 'path'
+
+const entry = [
+  'src/index.ts',
+  'src/actions.ts',
+  'src/chains.ts',
+  'src/connectors/coinbaseWallet.ts',
+  'src/connectors/injected.ts',
+  'src/connectors/metaMask.ts',
+  'src/connectors/mock.ts',
+  'src/connectors/walletConnect.ts',
+  'src/providers/alchemy.ts',
+  'src/providers/public.ts',
+  'src/providers/infura.ts',
+  'src/providers/jsonRpc.ts',
+]
+
+export default defineConfig(() => {
```

Comment:
Thoughts on moving this config up into a `tsup.config.ts` on the root, and then passing through the entries inside this file?

```tsx
import { getConfig } from '../../utils/tsup.ts'

const entry = [
  'src/index.ts',
  'src/actions.ts',
  'src/chains.ts',
  'src/connectors/coinbaseWallet.ts',
  'src/connectors/injected.ts',
  'src/connectors/metaMask.ts',
  'src/connectors/mock.ts',
  'src/connectors/walletConnect.ts',
  'src/providers/alchemy.ts',
  'src/providers/public.ts',
  'src/providers/infura.ts',
  'src/providers/jsonRpc.ts',
]

export default getConfig({ entry })
```

will be good if we have more packages down the line

Reply:
good idea! added